benchmarks/kernels/benchmark_paged_attention.py

```diff
@@ -101,7 +101,7 @@ def run_cuda_benchmark(num_iters: int, profile: bool = False) -> float:
         start_time = time.perf_counter()
 
         # Using default kv_scale
-        k_scale = v_scale = 1.0
+        k_scale = v_scale = 0.1
```

Comment:
also, can you explain the default kv-scale change?

Reply:
We are also unsure about these changes in the parameters set by the original author. Is it possible to get in touch with the author to understand about his optimization and algorithm?